src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs

```diff
@@ -22,7 +22,9 @@ public static class ResNet18Extension
         /// </summary>
         public static EstimatorChain<ColumnCopyingTransformer> ResNet18(this DnnImageModelSelector dnnModelContext, IHostEnvironment env, string outputColumnName, string inputColumnName)
         {
-            return ResNet18(dnnModelContext, env, outputColumnName, inputColumnName, Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "DnnImageModels"));
+            // Substring is  removing initial "file:///" from the codebase string.
+            string executingAssemblyLocation = Directory.GetParent(Assembly.GetExecutingAssembly().CodeBase.Substring(8)).FullName;
```

Comment:
There are a few other places we are calling `GetExecutingAssembly`

https://github.com/dotnet/machinelearning/search?q=GetExecutingAssembly&unscoped_q=GetExecutingAssembly

Do those places need to be fixed as well? Especially the other `DnnImageFeaturizer` places.

Reply:
I corrected it for other ```DnnImageFeaturizer``` places.
We can change at other places when we add tests for them.